src/component/events/EventCard/EventCard.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { makeStyles } from 'tss-react/mui';
```

Comment:
Trying to deprecate this use since it's an external library and no longer maintained by Mui. Can we use the styled components approach instead here?